packages/rest-hooks/src/react-integration/__tests__/useCache.tsx

```diff
@@ -4,8 +4,7 @@ import {
   PaginatedArticleResource,
 } from '../../__tests__/common';
 import { useCache } from '../hooks';
-import makeRenderRestHook from '../../test/makeRenderRestHook';
-import { makeCacheProvider } from '../../test/providers';
+import { makeRenderRestHook, makeCacheProvider } from '../../../../test/src';
```

Comment:
haha so many more layers now

Reply:
yeah would be cool if you could reference the test package without a relative import

Reply:
i suppose i could add another path alias, but i'd rather try to keep that as clean as possible for now